### PR TITLE
Nguyen Anh Tuan week 7

### PR DESCRIPTION
**Pull request by NguyenAnhTuan-20194201**
Nguyen Anh Tuan week 7

### CODE REVIEW

**Review by haitd-vbee — COMMENTED** [review]
Mất báo cáo của bạn 20204510 rồi

File: week-7/20194201/Server_SV/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+DB_CONNECTION= mongodb+srv://SV:sv123456789@cluster0.vp0j8zy.mongodb.net/?retryWrites=true&w=majority
```
Comment (haitd-vbee) [inline week-7/20194201/Server_SV/.env:1]: Add a new line at end of file

File: week-7/20194201/Server_SV/app.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    res.status(200).json({
+        message: `Welcome to Server_SV, ${statusDB}`,    
+          
+    });
+
+});
+
+app.listen(PORT,(req, res)=>{
+    console.log(`Server is live on ${PORT}`);
+});
```
Comment (haitd-vbee) [inline week-7/20194201/Server_SV/app.js:43]: Add a new line at end of file

File: week-7/20194201/Server_SV/controllers/student.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
+const mongoose = require("mongoose");
+const multer = require("multer")
+const studentModel = require("../models/student");
+const studentController ={};
+
+studentController.addStudent = (req, res,next) =>{
```
Comment (haitd-vbee) [inline week-7/20194201/Server_SV/controllers/student.js:6]:
```suggestion
studentController.addStudent = (req, res, next) =>{
```

File: week-7/20194201/Server_SV/models/student.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    avatar:{
+        data: Buffer,
+        type: String
+
+    }
+
+
+});
+
+module.exports = mongoose.model('student', studentSchema);
```
Comment (haitd-vbee) [inline week-7/20194201/Server_SV/models/student.js:41]: Add a new line at end of file

File: week-7/20194201/Server_SV/routes/index.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+
+const apiRoutes = experess.Router();
+apiRoutes.post("/postStudent",upload.single('image'),studentController.addStudent);
+apiRoutes.get("/getAllStudent",studentController.getAllStudent);
+apiRoutes.get("/findStudentByMssv/:mssv", studentController.getStudent);
+apiRoutes.get("/findStudentByName/:name", studentController.getStudent);
+apiRoutes.put("/updateStudentId/:id", studentController.updateStudent);
+apiRoutes.delete("/deleteStudentId/:id",studentController.deleteStudent);
+module.exports =  apiRoutes;
```
Comment (haitd-vbee) [inline week-7/20194201/Server_SV/routes/index.js:33]: Add a new line at end of file

File: week-7/20194201/Server_SV/routes/index.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+const experess = require("express");
```
Comment (haitd-vbee) [inline week-7/20194201/Server_SV/routes/index.js:1]:
```suggestion
const express = require("express");
```

**Review by haitd-vbee — APPROVED** [review]

File: .gitignore [hunk 7]
```diff
@@ -1,5 +1,5 @@
 node_modules
 package-lock.json
-.env
+
```
Comment (thanhnx-vbee) [inline .gitignore:3]: add `.env `and `upload` in gitignore

**Review by thanhnx-vbee — CHANGES_REQUESTED** [review]

File: week-7/20194201/Server_SV/env [hunk 8]
```diff
@@ -0,0 +1 @@
+DB_CONNECTION = " "
```
Comment (thanhnx-vbee) [inline week-7/20194201/Server_SV/env:1]: DB_CONNECTION=

**Review by thanhnx-vbee — CHANGES_REQUESTED** [review]

**Review by thanhnx-vbee — APPROVED** [review]

**Pull request merged**